modules/javafx.media/src/main/java/javafx/scene/media/MediaPlayer.java

```diff
@@ -112,10 +112,11 @@
  * duration of media playback is then the product of the cycle duration and the
  * number of times the cycle is played. If the stop time of the cycle is reached
  * and the cycle is to be played again, the event handler registered with the
- * {@link #onRepeatProperty onRepeat} property is invoked. If the stop time is reached and
- * the cycle is <i>not</i> to be repeated, then the event handler registered
- * with the {@link #onEndOfMediaProperty onEndOfMedia} property is invoked. A zero-relative index of
- * which cycle is presently being played is maintained by {@link #currentCountProperty currentCount}.
+ * {@link #onRepeatProperty onRepeat} property is invoked. If the stop time is
+ * reached, then the event handler registered with the {@link #onEndOfMediaProperty onEndOfMedia}
+ * property is invoked regardless if the cycle is to be repeated or not.
```

Comment:
I recommend the following wording change:

`regardless if the cycle` --> `regardless of whether the cycle`

Reply:
Fixed.